.storybook/stories/Heading.stories.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
```

Comment:
This story is here to show the default HTML element styles of headings in Decanter.